### ISSUE TITLE
errors not being returned properly

### ISSUE THREAD

**Issue by ebuchman** [eris-ltd/eris-compilers#55]
compile a contract with a syntax error via `eris-compilers compile abc.sol` and get `<nil>`

**Comment by VoR0220** [comment]
Can you show me some of the source code for the contract please?

**Comment by ebuchman** [comment]
paste anything at all into a file and run the command above! eg.

```
contract MyContract{
	fail
}
```

The output is simply `nil`. If I rollback the git history to 1967c4ffe1870416b67c6d6046102a71e5ce34f3 it works fine (ie. gives me a nice error message: 

```
exit status 1; 1f40c6b37d2adef8367b51ce675ed8b0585613931fc00ecb74cdf57fc3b3782d.sol:3:1: Error: Expected identifier, got 'RBrace'

```

**Comment by compleatang** [comment]
This is because we've changed the request/response structs between the server and client. Will fix before 0.11.4
